packages/relay-service/lib/index.js

```diff
@@ -35,7 +36,7 @@ const sendMessage = async ({keyPairDoc, recipientDid, message}) => {
 
   const {payload, did} = await generateSignedPayload(keyPairDoc, {
     to: recipientDid,
-    msg: JSON.stringify(jweMessage),
+    msg: toBase64(jweMessage),
```

Comment:
why is this change neededs? stringified json should work just fine. from the api:
```
this is incorrect, see on the API:

```
  const { payload, did } = await generateSignedPayload(keyPairDoc, {
    to: recipientDid,
    msg: typeof message === 'string' ? message : JSON.stringify(message),
  });
```